packages/sales-api-service/src/services/renewals/__tests__/renewals.service.spec.js

```diff
@@ -28,7 +52,8 @@ describe('preparePermissionDataForRenewal', () => {
       },
       shortTermPreferredMethodOfConfirmation: {
         label: 'Text'
-      }
+      },
+      noLicenceRequired: true
```

Comment:
Feels kinda weird applying this as `true` by default when in practice it should only be true on junior licences. If we have a test for juniors then I'd rather override it then and leave it false the rest of the time.